### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by obibaadoma**
Setup a new React application 
Create a directory for your reusable components: /components.
Install [React Router V6]
[Setup your router]

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @obibaadoma ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly ensure your project is using the correct GitFlow. Make sure your `feature branch` is merging into the `development branch` instead of the `main branch`.
- [ ] Kindly ensure your linter checks are passing and are in green. Currently, your tests are not passing and are in red.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,168 @@
...
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
```
Comment (Efiamotu-1) [inline README.md]: - [ ] please kindly remove this texts as its sole purpose is to help you customize this section 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,168 @@
...
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (Efiamotu-1) [inline README.md]: - [ ] please kindly remove this texts as its sole purpose is to help you customize this section 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,168 @@
...
+- GitHub: (https://github.com/obibaadoma)
+- Twitter: (https://twitter.com/obibaadoma)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/derek-akrasi-konadu)
+
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (Efiamotu-1) [inline README.md:114]: - [ ] please kindly remove this texts as its sole purpose is to help you customize this section 👍

File: README.md [hunk 4]
```diff
@@ -1 +1,168 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (Efiamotu-1) [inline README.md:136]: - [ ] please kindly remove this texts as its sole purpose is to help you customize this section 👍

File: README.md [hunk 5]
```diff
@@ -1 +1,168 @@
...
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author**
+
+- GitHub: (https://github.com/obibaadoma)
+- Twitter: (https://twitter.com/obibaadoma)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/derek-akrasi-konadu)
```
Comment (Efiamotu-1) [inline README.md]: - [ ] This link is currently broken please try to use the appropriate link to your LinkedIn account as it is not appropriate to leave a broken link In your readme 👍

File: README.md [hunk 6]
```diff
@@ -1 +1,168 @@
...
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (Efiamotu-1) [inline README.md]: - [ ] please kindly remove this texts as its sole purpose is to help you customize this section 👍

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import CategoryPage from './components/Category';
+
+const App = () => {
+  const dispatch = useDispatch();
+  dispatch(fetchBooks());
+  return (
+    <div>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookList />} />
```
Comment (Efiamotu-1) [inline src/App.js]: - [x] You are required to have a  `<Books>` component, which contains the following components: `<BookList>` which renders a list of books using the `<Book>` component. Following this structure, it is the `Books` component that should be imported into the highlighted piece of code. 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @obibaadoma , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (obibaadoma) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (obibaadoma) [reply README.md]: Removed

File: README.md (hunk 4 above)
Comment (obibaadoma) [reply README.md:136]: Removed

File: README.md (hunk 5 above)
Comment (obibaadoma) [reply README.md]: Link fixed

File: README.md (hunk 6 above)
Comment (obibaadoma) [reply README.md]: Removed

File: src/App.js (hunk 7 above)
Comment (obibaadoma) [reply src/App.js]: fixed

**Review by aamirkhan2478 — APPROVED** [review]
Hi <STUDENT_MENTION>, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**